src/target/riscv/riscv.c

```diff
@@ -3797,6 +3897,13 @@ static const struct command_registration riscv_exec_command_handlers[] = {
 		.usage = "set [vs] [vu] [m] [s] [u] <exception_codes>|clear",
 		.help = "Set or clear a single exception trigger."
 	},
+	{
+		.name = "icount",
```

Comment:
For clarity, I'd propose a more descriptive name:  `icount_trigger`

Reply:
How strongly do you feel about that? `icount_trigger` is such a lot to type, and the other trigger types don't have _trigger appended because the names already contain `trigger`. (The debug spec naming is inconsistent in this regard.)

Reply:
I definitely do not feel strongly about this one.

Generally, I tend to prefer clarity over short names. If I imagine I don't know this command and need to study an OpenOCD's .cfg file written by someone else where it is used, I'd like to see there `riscv icount_trigger set <...>` rather than the shorter form (`riscv icount set <...>`). The former immediately tells me to what is going on and where to look for details (that is, the "Triggers" chapter in the RISC-V Debug Spec).

Reply:
If you don't feel strongly, I'll just say that the naming is this way to be consistent with the debug spec and the existing itrigger and etrigger commands.

If someone out there wants to weigh in on Jan's side, I'll happily change the name.